### PR TITLE
feature: Nix search

### PR DESCRIPTION
**Pull request by bketelsen**
If merged this PR adds the ability to search nixpkgs from apx.

### CODE REVIEW

**Comment by kbdharun** [comment]
Wow, wow wow finally

**Review by kbdharun — APPROVED** [review]

File: cmd/upgrade.go [hunk 1]
```diff
@@ -68,3 +66,14 @@ func upgrade(cmd *cobra.Command, args []string) error {
...
+func upgradePackage(cmd *cobra.Command, args []string) error {
+
+	err := core.NixUpgradePackage(args[0])
+	if err != nil {
+		return err
+	}
+	cmdr.Success.Println(apx.Trans("nixupgrade.success"))
+	return nil
+
+}
```
Comment (matbme) [inline cmd/upgrade.go:79]: Isn't this part of the other PR? I think you added the changes from #122 to this one by accident.

Comment (bketelsen) [reply cmd/upgrade.go:79]: yes, but if we merge the other first my senior citizen moment won't matter.

**Comment by RDKRACZ** [comment]
```
apx --nix search htop
error: experimental Nix feature 'nix-command' is disabled; use '--extra-experimental-features
```

Is it not working correctly or am I doing something wrong?

**Comment by matbme** [comment]
Did you run`apx --nix init` before searching?

**Comment by matbme** [comment]
Also, can you check ` .config/nix/nix.conf` exists and `nix-command` is there?

**Comment by RDKRACZ** [comment]
Thank you, it's now working correctly after the init subcommand

**Pull request merged**